### PR TITLE
Feature/mp4 parser

### PR DESCRIPTION
**Pull request by dadurex**
plz merge

### CODE REVIEW

File: metaparser/modules/mp4.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        if field not in self.get_fields():
+            raise KeyError("Field not present in parser")
+        self.__file.tags[field] = value
+
+    def clear(self):
+        self.__file.tags.clear()
+
+    def delete_field(self, field: str) -> None:
+        if field not in self.get_fields():
+            raise KeyError("Field not present in parser")
```
Comment (RouNNdeL) [inline metaparser/modules/mp4.py]: This is part of the `super` class. Call super

File: metaparser/modules/mp4.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            FIELD_MUSICBRAINZ_ALBUMTYPE,
+            FIELD_RELEASECOUNTRY,
+            FIELD_BPM,
+            FIELD_TRACKNUMBER,
+            FIELD_DISCNUMBER,
+        ]
+
+    def set_field(self, field: str, value: Any) -> None:
+        if field not in self.get_fields():
+            raise KeyError("Field not present in parser")
```
Comment (RouNNdeL) [inline metaparser/modules/mp4.py]: This is part of the `super` class. Call super

**Review by RouNNdeL — CHANGES_REQUESTED** [review]

File: metaparser/modules/mp4.py (hunk 1 above)
Comment (dadurex) [reply metaparser/modules/mp4.py]: Added

File: metaparser/modules/mp4.py (hunk 2 above)
Comment (dadurex) [reply metaparser/modules/mp4.py]: Added

**Comment by MBWlodarczyk** [comment]
nice code dadurex

**Review by RouNNdeL — APPROVED** [review]

**Pull request merged**